sopel/config/core_section.py

```diff
@@ -69,9 +69,17 @@ class CoreSection(StaticSection):
 
     auth_method = ChoiceAttribute('auth_method', choices=[
         'nickserv', 'authserv', 'Q', 'sasl', 'server', 'userserv'])
-    """The method to use to authenticate with the server.
+    """Legacy method to authenticate with the server.
```

Comment:
I feel like many (most?) people just use a single auth method, probably nick-based auth (like nickserv).  So it seems like things should remain backwards-compatible, and keep the auth_method?

Maybe it would be better to add *only* the nick_auth_* params, and just add documentation saying the existing auth_method can be used for either server-based or nick-based auth (just like before) if you only have a single auth method; but if you want to do both server-based and nick-based auth, then use "auth_method" for server auth, and "nick_auth_method" for nick-based auth?

I can update the pull req if you think that's a good idea?

Reply:
I think it's OK to keep `auth_method` and both `nick_*`/`server_*`, but I'd prefer see "Simple authentication method" instead of "Legacy".

Legacy is scary. Simple is good. :smile:

Reply:
Also let's ping @dgw on that, maybe 3 brains are better than 2.

Reply:
> Legacy is scary. Simple is good. 😄

This, lol.

Reply:
ok, latest pull req has 'Legacy' changed to 'Simple' - no other changes.